src/engine/cuecontrol.cpp

```diff
@@ -219,8 +215,9 @@ void CueControl::trackLoaded(TrackPointer pNewTrack) {
             pLoadCue = pCue;
         }
         int hotcue = pCue->getHotCue();
-        if (hotcue != -1) {
-            attachCue(pCue, hotcue);
+        HotcueControl* pControl = m_hotcueControls.value(hotcue);
```

Comment:
Ok, the *defaultValue* is defined:
https://doc.qt.io/qt-5/containers.html#default-constructed-value
By omitting it we avoid confusion.